lib/ae_mdw/db/sync/origin.ex

```diff
@@ -0,0 +1,35 @@
+defmodule AeMdw.Db.Sync.Origin do
+  @moduledoc """
+  Generates a list of Origin related mutations to map a transactions object/pubkey to a txi and inversely too.
+  """
+
+  alias AeMdw.Node
+  alias AeMdw.Node.Db, as: NodeDb
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.Mutation
+  alias AeMdw.Db.MnesiaWriteMutation
+  alias AeMdw.Db.WriteFieldMutation
+  alias AeMdw.Txs
+
+  require Model
+
+  # @type origin_mutations :: [MnesiaWriteMutation.t() | WriteFieldMutation.t()]
+
+  @spec origin_mutations(
+          Node.tx_type(),
+          WriteFieldMutation.pos(),
+          NodeDb.pubkey(),
+          Txs.txi(),
+          Txs.tx_hash()
+        ) :: Mutation.t()
```

Comment:
Not sure why it didn't fail, but this should be a list of mutations, not a single mutation

Reply:
finally a dialyzer true error
fixed